doc/source/mast_data_products/hap_tests.rst

```diff
@@ -0,0 +1,195 @@
+Regression Tests
+================
+.. _regression-tests:
+
+
+**SVM**
+
+Several HAP specific tests do various checks on different datasets included in the test file name (e.g. test_svm_wfc3ir.py). 
+The four groups of tests used are as follows:
+
+A. Simple tests:
+   
+    test_svm_samewcs
+     * Checks that products for both detectors are aligned to the same catalog.
+
+    test_svm_wcs
+     * Checks the output primary WCSNAME includes FIT_SVM_GAIA.
+
+    test_svm_manifest_name
+     * Ensures the manifest file is created.
+
+   This also includes the general success of SVM alignment and catalog creation code.
+
+
+B. SVM catalog tests (currently disabled)
+    
+    test_svm_point_total_cat
+     * Tests the number of sources in the point source catalog compared to expected value. 
+    
+    test_svm_segment_total_cat
+     * Tests the number of sources in the segmentation source catalog compared to expected value. 
+
+
+C. Tests for different detectors
+    
+    test_svm_wcs_ir, test_svm_wcs_ir_all, test_svm_wcs_uvis, test_svm_wcs_uvis_all
+     * SVM alignment and catalog creation for different detectors.
+
+
+D. Mean magnitude tests
+
+    test_svm_point_cat_meanmag, test_svm_segment_cat_meanmag: 
+     * Checks that catalog mean magnitudes are within 0.5% of previous values.
+
+
+test_svm_je281u.py: A
+
+test_svm_hrcsbc.py: A, B
+
+test_svm_wfc3ir.py: A, B
+
+test_svm_j97e06.py: A, D
+
+test_svm_ibqk07.py: A, B, C, D
+
+test_svm_ibyt50.py: A, B, C
+
+|
+
+**HAP**
+
+test_processing_utils.py
+
+    test_add_skycell_to_header
+     * Unit test for function for adding skycell name to SVM headers.
+
+test_pipeline.py
+
+    test_astrometric_singleton
+     * A test of runastrodriz.process with varying environment setups. Obtains data using aqutils.retrieve_observations.
+
```

Comment:
_aqutils_ is only a short-hand name used in this file.  Write out the full module name which is _astroquery_utils_.